nhais/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+tornado = "*"
+pycurl = {version = "~=7.43",platform_system = "!= 'Windows'"}
```

Comment:
I noticed that @jaroslaw-wojciechowski also commented on this. Do we even need pycurl in the packages? Could we just remove this line?

Reply:
We need it for tornado